pkg/cmd/objects/browse/browse.go

```diff
@@ -68,6 +69,10 @@ func NewBrowseCmd(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 
+	cmd.SetUsageFunc(func(cmd *cobra.Command) error {
+		return cmdutil.UsageFuncWithInheritedFlagsOnly(f.IOStreams, cmd)
```

Comment:
Could we wrap this function so we can do `cmd.SetUsageFunc(cmdutil.UsageFuncWithInheritedFlagsOnly(...))` ?

Reply:
It's better I agree!